lib/login_gov/hostdata/railtie.rb

```diff
@@ -0,0 +1,34 @@
+begin
+  require 'lograge'
+rescue LoadError => e
+  STDERR.puts 'lograge is required by the LoginGov::Hostdata::Railtie, please add it to the Gemfile'
+  raise e
+end
```

Comment:
This is... annoying and weird. To make sure it worked correctly, I tried booting the dashboard app with this branch, but without `lograge` and it blew up with a long stacktrace and printed this message as expected.

The alternative is to add `lograge` as a hard dependency of this gem, which ends up pulling in these gems to every place we use this gem:
- activesupport
- actionpack
- railties
- request_store

*Most* of the time we use identity-hostdata we're inside a rails app, so it may not be that bad? But we do use this gem outside of rails:
- [in tests for identity-idp-config](https://github.com/18F/identity-idp-config/search?q=hostdata)
- [in identity-saml-sinatra](https://github.com/18F/identity-saml-sinatra/search?q=hostdata)... which may not be needed since we usually deploy to cloud.gov and not our own infra?

Open to feedback here!

Reply:
Curious also: How much does Lograge give us, vs. implementing our own [`ActiveSupport::LogSubscriber`](https://api.rubyonrails.org/classes/ActiveSupport/LogSubscriber.html)?

Reply:
Is this potentially out of scope for this gem? If it's used in non-rails contexts, and the logging dependency brings rails in, should we explore a different avenue?

Reply:
I agree that configuring the logging of the application feels like a different mission that providing information about the deployed environment. Perhaps this goes in `identity-logging` and is included by our rails apps only?

Reply:
> Is this potentially out of scope for this gem? If it's used in non-rails contexts, and the logging dependency brings rails in, should we explore a different avenue?

> I agree that configuring the logging of the application feels like a different mission that providing information about the deployed environment. Perhaps this goes in identity-logging and is included by our rails apps only?

Ok I am convinced, time for a new gem

> Curious also: How much does Lograge give us, vs. implementing our own [`ActiveSupport::LogSubscriber`](https://api.rubyonrails.org/classes/ActiveSupport/LogSubscriber.html)?

Potentially we could write our own, the thing I like about lograge is that it also quiets a bunch of extra Rails logging (like view rendering) that really distracts, and keeps that simple.